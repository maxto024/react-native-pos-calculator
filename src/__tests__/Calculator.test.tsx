import { shallow } from 'enzyme'
import * as React from 'react'
import { NativeModules } from 'react-native'
import { Calculator } from '../Calculator'

NativeModules.UIManager = {
  measureLayoutRelativeToParent: jest.fn()
}

it('renders calculator correctly', () => {
  const wrapper = shallow(<Calculator />)
  expect(wrapper).toMatchSnapshot()
})
