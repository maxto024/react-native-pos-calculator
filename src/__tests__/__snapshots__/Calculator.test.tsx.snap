// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders calculator correctly 1`] = `
<Component
  onLayout={[Function]}
/>
`;
